jest.config.js

```diff
@@ -1,6 +1,11 @@
 module.exports = {
   preset: "ts-jest",
-  testEnvironment: "jsdom",
+  testEnvironment: "jest-environment-jsdom",
   watchPlugins: ["jest-watch-typeahead/filename", "jest-watch-typeahead/testname"],
   testTimeout: 30000,
+  transformIgnorePatterns: ["node_modules/(?!swiper|@govtechsg|uuid).*/"],
+  transform: {
+    "^.+\\.ts?$": "ts-jest",
+    "^.+\\.(js|jsx)$": "babel-jest",
+  },
```

Comment:
would just using babel.config.json let you skip the need for transforming the js files?